File: waktu_jam_menit_detik2.py
class Jam:
  def __init__(self, hh, mm, ss):
      self.hh = hh
      self.mm = mm
      self.ss = ss
      
def pad_zero(n):
  return '0' + str(n) if n < 10 else str(n)

print('Masukkan total detik: ', end='')
totalDetik = int(input())

j = Jam(totalDetik // 3600, (totalDetik % 3600) // 60, (totalDetik % 3600) % 60)

print()
print('Waktu: ' + pad_zero(j.hh) + ':' + pad_zero(j.mm) + ':' + pad_zero(j.ss))

File: WaktuJamMenitDetik2.java
import java.util.Scanner;

public class WaktuJamMenitDetik2 {
    public static void main(String args[]) {
        Scanner in = new Scanner(System.in);

        System.out.print("Masukkan total detik: ");
        int totalDetik = in.nextInt();

        Jam j = new Jam(totalDetik / 3600, (totalDetik % 3600) / 60, (totalDetik % 3600) % 60);

        System.out.println();
        System.out.println("Waktu: " + padZero(j.hh) + ":" + padZero(j.mm) + ":" + padZero(j.ss));

        in.close();
    }

    public static String padZero(int n) {
        return n < 10 ? "0" + n : "" + n;
    }
}
